app/controllers/admin/roles_controller.rb

```diff
@@ -2,11 +2,13 @@ module Admin
   class RolesController < Admin::BaseController
     load_and_authorize_resource :conference, find_by: :short_title
     before_action :set_selection
+    authorize_resource :role
     # Show flash message with ajax calls
     after_action :prepare_unobtrusive_flash, only: :toggle_user
 
     def index
       @roles = Role.where(resource: @conference)
+      authorize! :index, @roles
```

Comment:
Doesn't `authorize_resource :role` already cover `index` action? Why do we need it here too?